src/components/site-content-tabs.tsx

```diff
@@ -24,7 +24,7 @@ export function SiteContentTabs() {
 		);
 	}
 
-	if ( selectedSite?.isAddingSite ) {
+	if ( selectedSite?.isAddingSite || selectedSite?.importState === 'new-site-importing' ) {
```

Comment:
This looks strange but it is necessary to be able to differentiate if the site is importing for the first time or not. This differentiation helps us to ensure that the loading state does not break on the `Import/ Export` tab and we can use the distinct loading state for the first time import which is what we have in the design.